src/index.html

```diff
@@ -1,13 +1,271 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBike</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="shortcut icon"
+      href="./styles/images/icons/favicon.png"
+      type="image/x-icon"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img
+            src="styles/images/logo-whitex2.png"
+            alt="mybike logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a
```

Comment:
Try to use `svg` for icons for better quality in future.
<img width="578" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/28480057/6a3aa039-f131-4bae-ad8f-89197c8c56eb">